### PR TITLE
Brayns release 0.8.0

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

File: bbp/viz/brayns/default.nix [hunk 1]
```diff
@@ -30,29 +38,43 @@ assert restInterface -> (rockets != null);
...
 
 
-	cmakeFlags = [ 
-			"-DCOMMON_DISABLE_WERROR=TRUE"
+	cmakeFlags = [
+                        "-DCOMMON_DISABLE_WERROR=TRUE"
+			"-DBRAYNS_OPTIX_TESTS_ENABLED=OFF"
+			"-DGIT_REVISION=e3b3e33f"
 			"-DOSPRAY_ROOT=${ospray}"
+			"-DBRAYNS_STEREOSCOPY_ENABLED=TRUE"
```
Comment (tribal-tec) [inline bbp/viz/brayns/default.nix]: not needed

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Pull request merged**